Marlin/ultralcd.cpp

```diff
@@ -2697,7 +2697,9 @@ void kill_screen(const char* lcd_msg) {
       STATIC_ITEM(MSG_INFO_BAUDRATE ": " STRINGIFY(BAUDRATE), true); // Baud: 250000
       STATIC_ITEM(MSG_INFO_PROTOCOL ": " PROTOCOL_VERSION, true);    // Protocol: 1.0
       #ifdef POWER_SUPPLY
-        #if (POWER_SUPPLY == 1)
+        #if (POWER_SUPPLY == 0)
+          STATIC_ITEM(MSG_INFO_PSU ": Generic", true);  // Power Supply: ATX
```

Comment:
No. I have a generic PS that came with the printer. It is not ATX nor Xbox. There is no option to remotely turn it off.